### ISSUE TITLE
Running list of Common EHR data elements not included in CLIF-2.0

### ISSUE THREAD

**Issue by clif-consortium** [clif-consortium/CLIF#9]
- Respiratory Pattern
- Cardiac Rhythm 
- Retractions
- RT RS Secretion amount
- Cough
- Capnometry

**Comment by baanna23** [comment]
Code status - Full/partial/DNAR. Could be interesting Table 1 variable for a lot of overlapping studies.

**Comment by 08wparker** [comment]
@baanna23 very good point as code status is often used as exclusion criteria. I think we need a new table as code status will vary over time...

**`code_status`**
`hospitalization_id` 
`start_dttm` 
`code_status_name`
`code_status_category` with levels `c(DNR, UDNR, DNR/DNI, Full, Presume Full, Other)`

Creating this CDE will be very interesting

**Comment by sajor2000** [comment]
We need to make sure this syntax will capture all changes throughout
admission.

**Comment by 08wparker** [comment]
@sajor2000 yes I was envisioning a long table with every code status change recorded as a separate row. 

This should be lowered priority for now as it’s not on the critical path for ATS projects

**Comment by deetherese** [comment]
Discussed over CLIF WG meeting 04/30/2026 not relevant right now.
